CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ accidentally triggering the load of a previous DB version.**
 * #1648 Drop chunks for materialized hypertable
 * #1665 Add ignore_invalidation_older_than to timescaledb_information.continuous_aggregates view
 
+**Licensing changes**
+* Gapfill, reorder, and policies around reorder and drop chunks are
```

Comment:
gapfill is already community

Reply:
Not according to the code (see diff). 

I guess we could skip mentioning gapfill here if it was already announces as community previously.

Reply:
Gapfill requires community license and your code change only removes a warning and not an actual license check

Reply:
A warning includes a _check_ for a valid enterprise license , and the full message is: `"WARNING:  Timescale License expired"` - `"Your license expired on %s. Renew your license to continue using enterprise features."` Gapfill tests clearly show this warning.

The reason we often aren't enforcing is because we don't want to break the DB when the license expires and instead give the user a chance to renew it. This is true also for the move chunks functionality, which is an enterprise feature that only raises a warning without enforcement (not my addition).

With this in mind, maybe you can point out what you'd like to see changed because I am not actually sure what you are trying to say.

I think you are saying that gapfill is community licensed but the code was never updated to reflect that? If that's the case, I think we should still make a note of this in the changelog because I guess that some gapfill users could have been mislead to believe that they couldn't use gapfill because of the warnings.

Reply:
The warning is a generic license expiration warning and not specific to enterprise.